manifests/08-prometheus_rule.yaml

```diff
@@ -20,4 +20,13 @@ spec:
       for: 5m
       labels:
         severity: info
+    - alert: SCANotAvailable
+      annotations:
+        description: 'Simple content access is not enabled. See https://access.redhat.com/articles/simple-content-access to enable it.'
```

Comment:
I don't see how linking to this is helpful. This page doesn't talk about how to use SCA with OCP at all.

Reply:
If I follow the controller logic correctly insights sets this up and establishes this thus this is simply an indication that the operator can't interact with OCM and subscription manager correctly: 

https://github.com/openshift/insights-operator/blob/master/pkg/ocm/ocm.go#L93-L136

Reply:
It's not about how to use it, but about how to enable it. Yes but there's an agreement with the OCM API that when we get HTTP 404 then it basically means that the user didn't enable the SCA subscription and thus the alert (it's only for `Reason=NotFound`).

Reply:
Anyway, I definetely welcome suggestions for a better description :-)